apps/web/src/components/Menu/GlobalSettings/SettingsModal.tsx

```diff
@@ -46,6 +45,8 @@ import {
 import { AtomBox } from '@pancakeswap/ui'
 import { useMMLinkedPoolByDefault } from 'state/user/mmLinkedPool'
 import styled from 'styled-components'
+import { TOKEN_RISK } from 'views/Swap/hooks/fetchTokenRisk'
```

Comment:
We better move the risk related constant and component to `src/components` if we want to use them in a common component.

When importing modules, we better keep in mind the dependency relationship between the modules.

Normally we shouldn't rely on a specific business component from a commonly used component. In this case from the import statement, it implies that the Menu component relies on Swap feature, which is not what we want and could cause further issue like circular reference

Reply:
done